### PR TITLE
Updated README for repo consistency project

### PR DESCRIPTION
**Pull request by bradleycamacho**
For context:
https://docs.google.com/document/d/1JIog8rJL9MT6_AbIji8VK_RT3mOuIxC2Xb_xQsVksKk/edit#

https://newrelic.atlassian.net/browse/DOC-6121

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -45,14 +41,87 @@ The ``NEW_RELIC_SERVICE_NAME`` environment variable can be set to customize the
...
+or identifiable individual, including, for example, your name, phone
+number, post code or zip code, Device ID, IP address, and email address.
+
+For more information, review `New Relic’s General Data Privacy
+Notice <https://newrelic.com/termsandconditions/privacy>`__.
+
+Contribute
+----------
+
+We encourage your contributions to improve [project name]! Keep in mind
```
Comment (a-feld) [inline README.rst:91]:
```suggestion
We encourage your contributions to improve newrelic-airflow-plugin! Keep in mind
```

**Review by a-feld — CHANGES_REQUESTED** [review]

File: README.rst [hunk 2]
```diff
@@ -45,14 +41,87 @@ The ``NEW_RELIC_SERVICE_NAME`` environment variable can be set to customize the
...
+hands with several support channels.
+
+If the issue has been confirmed as a bug or is a feature request, file a
+GitHub issue.
+
+**Support Channels**
+
+-  `New Relic Documentation <LINK%20to%20specific%20docs%20page>`__:
+   Comprehensive guidance for using our platform
+-  `New Relic Community <LINK%20to%20specific%20community%20page>`__:
```
Comment (a-feld) [inline README.rst:65]: Links are missing here

File: README.rst [hunk 3]
```diff
@@ -45,14 +41,87 @@ The ``NEW_RELIC_SERVICE_NAME`` environment variable can be set to customize the
...
+appreciate you reporting it to New Relic through
+`HackerOne <https://hackerone.com/newrelic>`__.
+
+If you would like to contribute to this project, review `these
+guidelines <./CONTRIBUTING.md>`__.
+
+To `all contributors <LINK%20TO%20contributors>`__, we thank you!
+Without your contribution, this project would not be what it is today.
+We also host a community project page dedicated to `Project
+Name <LINK%20TO%20https://opensource.newrelic.com/projects/...%20PAGE>`__.
```
Comment (a-feld) [inline README.rst:119]: Links are missing here.

File: README.rst [hunk 4]
```diff
@@ -45,14 +41,87 @@ The ``NEW_RELIC_SERVICE_NAME`` environment variable can be set to customize the
...
+
+License
+-------
+
+newrelic-airflow-plugin is licensed under the `Apache
+2.0 <http://apache.org/licenses/LICENSE-2.0.txt>`__ License. >
```
Comment (a-feld) [inline README.rst:125]: Looks like an extra `>` snuck in here.
```suggestion
2.0 <http://apache.org/licenses/LICENSE-2.0.txt>`__ License. 
```

**Comment by TimPansino** [comment]
@bradleycamacho As with the other repo I ping you on, can this PR be cleaned up so we can merge it, or would you rather close it?

**Comment by umaannamalai** [comment]
@bradleycamacho touching base with you on this PR as well! If you would like to make the requested changes, we can re-review or we can close this PR out.

**Pull request closed without merging**